invenio_theme/translations/tr/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,144 @@
+# Translations template for invenio-theme.
+# Copyright (C) 2020 CERN
+# This file is distributed under the same license as the invenio-theme
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
+# 
+# Translators:
+# Beraat Aldemir <berataldemir@gmail.com>, 2020
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: invenio-theme 1.3.0a10\n"
+"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
+"POT-Creation-Date: 2020-07-14 11:54+0300\n"
+"PO-Revision-Date: 2016-08-25 15:28+0000\n"
+"Last-Translator: Beraat Aldemir <berataldemir@gmail.com>, 2020\n"
+"Language-Team: Turkish (https://www.transifex.com/inveniosoftware/teams/23537/tr/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+"Language: tr\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: invenio_theme/ext.py:65
+msgid "Home"
+msgstr "Anasayfa"
+
+#: invenio_theme/templates/invenio_theme/401.html:13
+#: invenio_theme/templates/semantic-ui/invenio_theme/401.html:13
+msgid "Unauthorized"
+msgstr "Yetkisiz"
+
+#: invenio_theme/templates/invenio_theme/401.html:14
+#: invenio_theme/templates/semantic-ui/invenio_theme/401.html:14
+msgid "You need to be authenticated to view this page."
+msgstr "Bu sayfayı görüntüleyebilmek için giriş yapmış olmalısınız."
+
+#: invenio_theme/templates/invenio_theme/403.html:13
+#: invenio_theme/templates/semantic-ui/invenio_theme/403.html:13
+msgid "Permission required"
+msgstr "İzin gerekli"
+
+#: invenio_theme/templates/invenio_theme/403.html:14
+#: invenio_theme/templates/semantic-ui/invenio_theme/403.html:14
+msgid "You do not have sufficient permissions to view this page."
+msgstr "Bu sayfayı görüntüleyebilmek için yeterli izinlere sahip değilsiniz."
+
+#: invenio_theme/templates/invenio_theme/404.html:13
+#: invenio_theme/templates/semantic-ui/invenio_theme/404.html:13
+msgid "Page not found"
+msgstr "Sayfa bulunamadı"
+
+#: invenio_theme/templates/invenio_theme/404.html:14
+#: invenio_theme/templates/semantic-ui/invenio_theme/404.html:14
+msgid "The page you are looking for could not be found."
+msgstr "Aradığınız sayfa bulunamadı."
+
+#: invenio_theme/templates/invenio_theme/429.html:13
+#: invenio_theme/templates/semantic-ui/invenio_theme/429.html:13
+msgid "Too many requests"
+msgstr "Çok fazla istek"
+
+#: invenio_theme/templates/invenio_theme/429.html:14
+#: invenio_theme/templates/semantic-ui/invenio_theme/429.html:14
+msgid "You have made too many consecutive requests, please try again later."
+msgstr ""
+"Art arda çok fazla istekte bulundunuz, lütfen daha sonra tekrar deneyin."
```

Comment:
@baldemir Something seems strange here ( note the `msgstr ""` in the line `68`). Did you manage to check that this specific message translates properly? Maybe we need to change it to:
```suggestion
msgstr "Art arda çok fazla istekte bulundunuz, lütfen daha sonra tekrar deneyin."
```

Reply:
@zzacharo Yes similar to new line problem. I checked it, no problem about displaying the correct translation.

Reply:
Ok then!